app/move/controllers/create/base.js

```diff
@@ -24,6 +28,39 @@ class CreateBaseController extends FormWizardController {
 
     next()
   }
+
+  async setSummaryData(req, res, next) {
```

Comment:
`setMoveSummary`? Feels more descriptive of what it's setting. _Data_ is a bit ambiguous.